pkgs/top-level/all-packages.nix

```diff
@@ -170,6 +170,8 @@ in
 
   fetchMavenArtifact = callPackage ../build-support/fetchmavenartifact { };
 
+  homeassistant = python3Packages.homeassistant;
```

Comment:
if its an application and not a library then it shouldn't be in `python-packages.nix`. See below as well.